### ISSUE TITLE
Crashes in Java 16

### ISSUE THREAD

**Issue by VoxelDraqon** [toolbox4minecraft/amidst#960]
Crash in Java 16:
```
[crash] Amidst has encounted an uncaught exception on the thread Thread[AWT-EventQueue-0,6,main]
[crash] java.lang.reflect.InaccessibleObjectException: Unable to make void java.awt.Window.setGraphicsConfiguration(java.awt.GraphicsConfiguration) accessible: module java.desktop does not "opens java.awt" to unnamed module @5fb2de77
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:357)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
	at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:199)
	at java.base/java.lang.reflect.Method.setAccessible(Method.java:193)
	at amidst.gui.main.MultiMonitorFixer.getGCHandle(MultiMonitorFixer.java:34)
	at amidst.gui.main.MultiMonitorFixer.<init>(MultiMonitorFixer.java:28)
	at amidst.gui.main.PerMainWindowInjector.<init>(PerMainWindowInjector.java:73)
	at amidst.PerApplicationInjector.createMainWindow(PerApplicationInjector.java:101)
	at amidst.Application.displayMainWindow(Application.java:76)
	at amidst.gui.profileselect.LocalProfileComponent.loadFinished(LocalProfileComponent.java:156)
	at amidst.threading.WorkerExecutor.lambda$null$8(WorkerExecutor.java:72)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:117)
	at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:190)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:235)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:233)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:312)
	at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:233)
	at java.desktop/java.awt.Dialog.show(Dialog.java:1079)
	at java.desktop/javax.swing.JOptionPane.showOptionDialog(JOptionPane.java:874)
	at java.desktop/javax.swing.JOptionPane.showMessageDialog(JOptionPane.java:670)
	at java.desktop/javax.swing.JOptionPane.showMessageDialog(JOptionPane.java:641)
	at amidst.logging.AmidstMessageBox.lambda$displayMessageBox$0(AmidstMessageBox.java:107)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```
Warning in Java 15:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by amidst.gui.main.MultiMonitorFixer (file:/usr/share/java/amidst/AMIDST.jar) to method java.awt.Window.setGraphicsConfiguration(java.awt.GraphicsConfiguration)
WARNING: Please consider reporting this to the maintainers of amidst.gui.main.MultiMonitorFixer
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by moulins** [comment]
Fixed (kinda :c) in 7d0cbbe.

@burgerguy Your hack for multi-monitor setups unfortunately broke in Java 16, which is now required for Minecraft 21w19a and above. I've made a hotfix to simply swallow the error, but a better solution would be nice.

Do you have any ideas on how to replace with hack with something better? (other than requiring an `--add-opens` argument on the command line).

**Comment by burgerguy** [comment]
@moulins When I get home I'll see if I can figure out a better way, but the root problem is due to issues with AWT managing the graphics device. If there's a way that's supported by java 16 to do reflection stuff, then that will work. I also haven't checked to see if java 16 fixes the problem or not, because if it does then we can just disable it for java 16 and above.
